en/Getting started/Glossary.md

```diff
@@ -0,0 +1,96 @@
+This glossary includes common Obsidian terminology.
+
+## vault
+
+A **vault** is a folder on your file system with contains [[#note|notes]] and an `.obsidian` folder with Obsidian-specific configuration. See also [[How Obsidian stores data]].
+
+## note
+
+A **note** is a Markdown file inside a [[#vault]].
+
+## link
+
+A **link** references another note, file, or document. An [[Internal links\|internal link]] references a file inside a vault. An [[Basic formatting syntax#External links\|external link]] references a document outside the vault.
```

Comment:
what are the backslashes here for?

Reply:
Ah, good catch. I started out with a table and forgot to unescape the vertical lines :)